### PR TITLE
Update template to reflect new branding

### PR DESCRIPTION
**Pull request by kgcreative**
Signed-off-by: kgcreative <kvngar@amazon.com>

### Description
Updates email template to reflect new OpenSearch branding

Before:
Screen Shot 2021-07-15 at 6 41 45 PM

After:
Screen Shot 2021-07-15 at 7 06 57 PM

### Check List
- [ ] ~~New functionality includes testing.~~ 
  - [ ]~~All tests pass, including unit test, integration test and doctest~~ 
- [ ] ~~New functionality has been documented.~~ 
  - [ ] ~~New functionality has javadoc added~~ 
  - [ ] ~~New functionality has user manual doc added~~ 
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by zhongnansu — APPROVED** [review]
thx!

File: dashboards-reports/server/routes/utils/notification/notification_content_template/email_content_template.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
       body {
         background-color: #f6f6f6;
-        font-family: "Inter UI", -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+        font-family: "Open Sans", "Segoe UI", Tahoma, sans-serif;
```
Comment (joshuali925) [inline dashboards-reports/server/routes/utils/notification/notification_content_template/email_content_template.html:23]: just curious why did the font change? I see the previous `font-family` combination in dashboards a lot

**Review by joshuali925 — APPROVED** [review]
looks good, thanks

**Review by davidcui-amzn — APPROVED** [review]

File: dashboards-reports/server/routes/utils/notification/notification_content_template/email_content_template.html (hunk 1 above)
Comment (kgcreative) [reply dashboards-reports/server/routes/utils/notification/notification_content_template/email_content_template.html:23]: Couple of reasons. The new brand uses Open Sans, and becuase this is email, this fits the brand. The second reason is that this new font stack is available in more systems out of the box. We're not really using emojis and symbols in the email, so those are redundant. Inter UI is not a common system font.

Comment (joshuali925) [reply dashboards-reports/server/routes/utils/notification/notification_content_template/email_content_template.html:23]: got it, thanks

**Pull request merged**